tests/compatibility_tests/create_index/test_create_index.py

```diff
@@ -0,0 +1,36 @@
+from sys import exc_info
+
+import pytest
+
+from tests.compatibility_tests.base_test_case.base_compatibility_test import BaseCompatibilityTestCase
+
+@pytest.mark.marqo_version('2.0.0')
+class TestCreateIndex(BaseCompatibilityTestCase):
+    index_name = "test_create_index_api"
+
+    @classmethod
+    def tearDownClass(cls) -> None:
+        cls.indexes_to_delete = [cls.index_name]
+        super().tearDownClass()
+
+    def prepare(self):
+
+        all_results = {}
+        try:
+            self.logger.debug(f"Creating index {self.index_name}")
+            self.client.create_index(index_name = self.index_name)
```

Comment:
You're not testing structured

Reply:
Actually saw the other one. Can we name this unstructured?

Reply:
I want to stick to API names to lower confusion